api/end_device.proto

```diff
@@ -570,6 +570,8 @@ message EndDevice {
   // The field cannot be unset once set.
   google.protobuf.Timestamp activated_at = 53;
 
+  google.protobuf.Timestamp last_seen = 54;
```

Comment:
A short description would help, and this probably should be `last_seen_at` (everything ends in `_at` for timestamps in this proto).